src/test/java/tests/EnikshayEndOfFormTests.java

```diff
@@ -0,0 +1,95 @@
+package tests;
+
+import beans.NewFormResponse;
+import beans.SubmitResponseBean;
+import beans.menus.CommandListResponseBean;
+import beans.menus.EntityDetailListResponse;
+import beans.menus.EntityListResponse;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import utils.TestContext;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+
+/**
+ * Regression tests for fixed behaviors
```

Comment:
Oh interesting I had thought this was purely a refactor; what issues did it fix?

Reply:
I'm going to have to get better at my class comments with you reviewing...